lib/mindwendel_web/live/brainstorming_live/show.html.leex

```diff
@@ -67,7 +67,7 @@
       return_to: Routes.brainstorming_show_path(@socket, :show, @brainstorming) %>
   <% end %>
 
-  <%= live_component @socket, MindwendelWeb.IdeaLive.IndexComponent, ideas: @ideas, id: @brainstorming.id, current_user: @current_user %>
+  <%= live_component @socket, MindwendelWeb.IdeaLive.IndexComponent, ideas: @ideas, brainstorming: @brainstorming, id: @brainstorming.id, current_user: @current_user %>
```

Comment:
i guess we dont need the id parameter then. We dont need to pass both brainstorming + brainstorming_id